Cargo.toml

```
@@ -455,6 +455,13 @@ repository = "https://github.com/paritytech/polkadot-sdk.git"
 license = "GPL-3.0-only"
 version = "1.0.0"
 
+[profile.release]
+# Substrate runtime requires unwinding.
+panic = "unwind"
+
+[profile.production]
+inherits = "release"
```

```suggestion
[profile.production]
inherits = "release"
lto = true
codegen-units = 1
```

We should also delete the profiles from `polkadot/Cargo.toml` as those don't do anything anymore since that's not the root of the workspace anymore.